src/compressed_tensors/quantization/lifecycle/apply.py

```diff
@@ -189,14 +190,14 @@ def apply_quantization_config(
     return names_to_scheme
 
 
-def process_quantization_config(config: QuantizationConfig) -> QuantizationConfig:
+def process_quantization_config(config: Optional[QuantizationConfig]) -> Optional[QuantizationConfig]:
```

Comment:
Seems like the only use of `process_quantization_config` is by `apply_quantization_config`.
What is the purpose of this change?